source/guides/references/protractor-migration-guide.md

```diff
@@ -0,0 +1,565 @@
+})
+```
+
+2. Set the inspector agent with a breakpoint flag and a config file
+3. Use Chrome's DevTools devices to locate the correct target 
+
+With Cypress however, because your tests are available through the browser dashboard, you can debug with DevTools without any additional configuration. Rather than rely solely on the `debugger` keyword, Cypress allows you to debug specific stages of your test by chaining the `debug()` command!
+
+**After: Cypress**
+
+```js
+describe('example test suite', () => {
+	it('contains an error we need to debug', () => {
+		cy.visit('/login')
+		cy.get('#password-field')).debug().sendKeys('testPassword1234')
+	})
+})
+```
+
+For more information, check out our [official documentation on debugging](https://docs.cypress.io/guides/guides/debugging.html#Using-debugger)!
+
+### Parallelization
+
+One of the worst things that can happen to a developer is to be forced to wait for a 2 hour end-to-end test suite to before verifying something works or not. Instead, Cypress Dashboard Service allows your tests to run in parallel. If your longest test only test a minute to run, this means that you've just cut down your testing by over 12,000%!
+
+This feature is available in Protractor and requires you to configure your application with multiple options (i.e., `sharedTestFiles`, `maxInstances`, etc.). An example is provided below:
+
+**Before: Protractor**
+
+```js
+// Example from https://developers.perfectomobile.com/display/PD/Protractor+parallel+execution
+// An example configuration file.
+exports.config = {
+
+  seleniumAddress: 'https://cloudName.perfectomobile.com/nexperience/perfectomobile/wd/hub',
+
+  // Capabilities to be passed to the webdriver instance.
+  capabilities: {
+    browserName: 'chrome',
+
+    // allows different specs to run in parallel.
+    // If this is set to be true, specs will be sharded by file
+    // (i.e. all files to be run by this set of capabilities will run in parallel).
+    // Default is false.
+    sharedTestFiles: true,
+
+    // Maximum number of browser instances that can run in parallel for this
+    // set of capabilities. This is only needed if shardTestFiles is true.
+    // Default is 1.
+    maxInstances: 2,
+
+    // Cloud capabilities
+    user: 'user@perfectomobile.com',
+    password: 'password',
+
+    // Device capabilities
+    platformName: 'Android',
+    manufacturer: 'Samsung',
+    model: 'Galaxy S5'
+  },
+
+  // Framework to use. Jasmine is recommended.
+  framework: 'jasmine',
+
+  // Spec patterns are relative to the current working directly when
+  // protractor is called.
+  specs: ['spec.js', 'spec2.js'],
+
+  // Options to be passed to Jasmine.
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000
+  }
+}
+```
+
+However, with Cypress, all you need to do is pass the `--parallel` and `--record` flag to `cypress run`, and it will take care of the rest for you:
+
+**After: Cypress**
+
+```bash
+$ cypress run --record --parallel
```

Comment:
Do not include `$` within terminal text - we want the text to be fully copy/pastable. Adding the `shell` as the language will add a `$` to the styles later (which will not be copied if user copies) and have the correct styling.

```suggestion
```shell
cypress run --record --parallel
```

**Before**

![Screen Shot 2020-05-21 at 3 34 31 PM](https://user-images.githubusercontent.com/1271364/82542845-aa136500-9b78-11ea-9603-2c8f77b68563.png)

**After**

![Screen Shot 2020-05-21 at 3 35 02 PM](https://user-images.githubusercontent.com/1271364/82542857-af70af80-9b78-11ea-8c07-ac6b1df30346.png)